### ISSUE TITLE
it wont build on windows 10

### ISSUE THREAD

**Issue by Kodi4444** [alexkirsz/dispatch#1]
i get this error
```
  = note: LINK : fatal error LNK1181: cannot open input file 'Packet.lib'


error: could not compile `dispatch-proxy` due to previous error
```

**Comment by Kodi4444** [comment]
well i got it to install but I had to find and download Packet.lib then put it in C:\Users\%Username%\.rustup\toolchains\stable-x86_64-pc-windows-msvc\lib\rustlib\x86_64-pc-windows-msvc\lib.

**Comment by gregyjames** [comment]
@Kodi4444 Where did you get packet.lib from?

**Comment by gregyjames** [comment]
Wait I found it. For anyone also stuck on the same issue you need to get the right packet.lib for your toolchain from [WinPcap 4.1.2 Developer's Pack](https://www.winpcap.org/devel.htm)

**Comment by Amir-m-a** [comment]
Same problem with win 11 + gnu toolchain (mingw64)
Fixed by installing [Npcap](https://npcap.com/#download) (which is recommended by WinPcap as it's not maintained anymore)
Packet.lib from sdk/lib/x64 as mentioned above fixed the build error
Had to install it too for packet.dll error
Works fine now

**Comment by alexkirsz** [comment]
Fixed in #2 (thanks @gregyjames!)
